packages/react/.storybook/webpack.config.js

```diff
@@ -78,6 +84,18 @@ module.exports = (baseConfig, env, defaultConfig) => {
     ],
   };
 
+  defaultConfig.module.rules.push({
+    test: /(\/|\\)FeatureFlags\.js$/,
```

Comment:
Flipping the switch upon React context is in consideration, for the time once we have finished implementing this mechanism to all form components.